docs/monitoring.md

```diff
@@ -253,10 +273,24 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Summary metrics of all tasks in the given stage attempt</td>
   </tr>
   <tr>
+    <td></td>
+    <td><code>?quantiles</code> summarize the metrics with the given quantiles.
+    Example: <br><code>?quantiles=0.01,0.5,0.99</code></td>
```

Comment:
In fact it is worse than that: the java8 javadoc engine will fail if a javadoc has an "illegal" closing tag in it. Which means that putting the closing tags in there breaks builds on java 8+. That's why I'm fairly cautious about closing the "classic" HTML tags these days.